<launch>
  <!-- SVO -->
  <node pkg="asr_sdm_ros2" type="svo_node" name="svo" clear_params="true" output="screen">
    <!-- Camera and IMU topics to subscribe to -->
    <param name="cam0_topic" value="/cam0/image_raw" type="str" />
    <param name="imu_topic" value="/imu0" type="str" />
    <param name="calib_file" value="$(find asr_sdm_ros2)/param/calib/euroc_mono.yaml" type="str" />

    <rosparam file="$(find asr_sdm_ros2)/param/pinhole.yaml" />
    <rosparam file="$(find asr_sdm_ros2)/param/frontend_imu/euroc_mono_imu.yaml" />

  </node>

  <!-- RVIZ + SVO GUI -->
  <node name="vis" pkg="rviz" type="rviz" args=" -d $(find asr_sdm_ros2)/rviz_config.rviz" />
  <node name="svo_gui" pkg="rqt_gui" type="rqt_gui" args="-s rqt_svo.svo.Svo --args --topic svo" />
</launch>
